qtrade/questrade.py

```diff
@@ -513,3 +513,168 @@ def submit_order(self, acct_id: int, order_dict: Dict[str, Union[int, bool, str]
         )
 
         return response
+
+    def lookup_symbol_by_name(self, symbolPrefix: str) -> List:
```

Comment:
I am wondering, what is the difference to `ticker_information`? Is is that you could search by company name here?

Reply:
Since I did not see ticker_information in my first commit, I removed `lookup_symbol_by_name` in favour of `ticker_information`